### DISCUSSION TITLE
Is it possible to divide a nwb file into channel subsets?

### DISCUSSION THREAD

**Discussion by alejoss30** [NeurodataWithoutBorders/helpdesk#76] [Q&A]
Hi,
I have electrophysiology recordings in .nwb format. Theres 64 channels of electrophysiology data in the file, however I need to access only a subset of them because I intend to do spike sorting processes separatedly in this two subsets of channels. Is it possible to leave the .nwb file with, lets say, only 16 of the original 64 channels? 

Thanks.

**Comment by CodyCBakerPhD** [comment]
attn @alejoe91 for spike sorting related question on NWB files

If he doesn't answer here, you might want to ask how to do this on https://github.com/SpikeInterface/spikeinterface

As far as my understanding goes, as long as they have the correct separation in the NWB file using different ElectrodeGroups for the respective channels, you can perform spike sorting 'by group' without splitting the file itself
